### PR TITLE
Aran ot errorfix

### PR DESCRIPTION
**Pull request by aranXmos**

### CODE REVIEW

File: experimental/xformer/Transforms/ConvPatternsTFL.cpp [hunk 1]
```diff
@@ -262,16 +262,16 @@ LogicalResult ReplaceConv2DPattern::getOutputTransformParams(
...
+    nn::OutputTransformFnInt8::get_quant_error(mulAndBiases, qp, args.quantErrorFullCheckEnabled);
   if (quantError > args.quantErrorThreshold) {
     // Try channelwise OT
     auto quantizer = nn::OutputTransformFnInt8_Channelwise::Quantizer();
     nn::OutputTransformFnInt8_Channelwise::QuantisationParams qp =
         quantizer.quantise_activation(mulAndBiases, false);
 
     quantError =
-        nn::OutputTransformFnInt8::get_quant_error(mulAndBiases, qp, false);
+        nn::OutputTransformFnInt8_Channelwise::get_quant_error(mulAndBiases, qp, false);
```
Comment (ghost) [inline experimental/xformer/Transforms/ConvPatternsTFL.cpp]: I think `false` indicates that quick check is being used?

File: experimental/xformer/Transforms/ConvPatternsTFL.cpp [hunk 2]
```diff
@@ -473,16 +473,16 @@ LogicalResult ReplaceDepthwiseConv2DPattern::getOutputTransformParams(
...
   if (quantError > args.quantErrorThreshold) {
     // Try channelwise OT
     auto quantizer = nn::OutputTransformFnInt8_Channelwise::Quantizer();
     nn::OutputTransformFnInt8_Channelwise::QuantisationParams qp =
         quantizer.quantise_activation(mulAndBiases, false);
 
-    quantError = nn::OutputTransformFnInt8_Channelwise::get_quant_error(
-        mulAndBiases, qp, false);
+    quantError = 
+      nn::OutputTransformFnInt8_Channelwise::get_quant_error(mulAndBiases, qp, false);
```
Comment (ghost) [inline experimental/xformer/Transforms/ConvPatternsTFL.cpp]: Same here?

Comment (aranXmos) [reply experimental/xformer/Transforms/ConvPatternsTFL.cpp]: you're right oops

**Review by ghost — APPROVED** [review]
+1

**Pull request merged**